src/utils/duration-humanizer.ts

```diff
@@ -0,0 +1,40 @@
+import * as humanizeDuration from 'humanize-duration';
+
+const getTimeInText = (seconds: number, uiConfig: any) => {
+  try {
+    const durationHumanizer = getDurationHumanizer(uiConfig);
+    if (durationHumanizer) {
+      try {
+        return seconds ? durationHumanizer(seconds * 1000, {round: true}) : '0';
```

Comment:
I added it because we don't want to see the numbers after the decimal point, only the integer number same as we see in the time on the bottom bar. but you are right regarding other places so i changed it to be an optional argument.